### PR TITLE
bump Jinja2 version

### PR DESCRIPTION
**Pull request by xiliuStrateos**
There is Jinja2 version conflicts when adding `transcriptic` as dependency to the `data-processor-service`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def run_tests(self):
         "requests>=2.0,<3",
         "pycryptodome==3.9.6",
         "python-magic>=0.4,<1",
-        "Jinja2>=2.0,<3",
+        "Jinja2>=3.0",
```
Comment (yangchoo) [inline setup.py:79]: we should bound it by <4 in case the next major version breaks this

File: transcriptic/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "9.4.2"
+__version__ = "9.4.3"
```
Comment (yangchoo) [inline transcriptic/version.py:1]: this is technically a feature bump and not a bugfix, so it sohuld be 9.5.0

Also, we usually separate the release commit from the feature commit.

**Review by yangchoo — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (xiliuStrateos) [reply setup.py]: Good point

**Review by yangchoo — APPROVED** [review]
thanks!

**Pull request merged**